cart-items.js

```diff
@@ -206,7 +206,7 @@ cart.get('/:id', (req, res) => {
 // c. Response Code: 201 (Created)
 cart.post('/', (req, res) => {
     //res.send('is it my code?');
-    let cart = [...myCart];
```

Comment:
`[...myCart]` is an example of a spread operator. This creates a clone of the original array, leaving the original unmodified.